client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js

```diff
@@ -65,7 +65,7 @@ const DataFilesProjectEditDescriptionModal = () => {
     title: Yup.string()
       .min(1)
       .required('Please enter a title.'),
-    description: Yup.string()
+    description: Yup.string().nullable()
```

Comment:
Also, might be simpler to set `description || ''` on [line 36](https://github.com/TACC/Frontera-Portal/blob/21242f5f7a78f613fce9878074d527d54bdc8b33/client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js#L36) above

Reply:
Nullable reflects the current default value better. Which would you rather?

Reply:
The change on [line 56](https://github.com/TACC/Frontera-Portal/pull/308/files#diff-0e7110cff4b6c26af79ab159523a70b791a726db19d890074442e88468c01a91R56) overrides the nullable anyway. Let's do empty string.

Reply:
Done.